.vscode/settings.json

```diff
@@ -3,5 +3,10 @@
     "workbench.colorCustomizations": {
         "activityBar.background": "#5a0656"
     },
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "search.exclude": {
+        "**/node_modules": true,
+        "**/bower_components": true,
+        "docs/**/*": true,
```

Comment:
Unrelated to PR, but noticed it added searching through `docs` was bad experience in VSCode.